laba 5 header

```diff
@@ -0,0 +1,105 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+#include <math.h>
+
+using namespace std;
+
+class RaceHorse {
+private:
+    string name;
+    int age;
+    double speed;
+    int place_in_race;
+public:
+    string get_name() const {
+        return name;
+    }
+    int get_age() const {
+        return age;
+    }
+    double get_speed() const {
+        return speed;
+    }
+    void set_place(int place) {
+        place_in_race = place;
+    }
+    double get_place() const {
+        return place_in_race;
+    }
+    RaceHorse(string name, double speed, int age) :name(name), speed(speed), age(age) {};
+};
+
+void print(RaceHorse horse) {
+    cout << "Name: " << horse.get_name() << " age: " << horse.get_age() << " speed: " << horse.get_speed() << endl;
+}
+
+bool operator == (const RaceHorse& h1, const RaceHorse& h2) {
+    return h1.get_name() == h2.get_name() && h1.get_age() == h2.get_age() && h1.get_speed() == h2.get_speed();
+}
+
+bool cmp_speed(RaceHorse horse1, RaceHorse horse2) {
```

Comment:
Function must start with capital letter. 
What is cmp_speed? This name is incomprehensible